### ISSUE TITLE
Why do you use FrozenBatchNorm2d?

### ISSUE THREAD

**Issue by 2006pmach** [researchmm/Stark#15]
Hi

I was wondering why you use `FrozenBatchNorm2d` in the ResNet backbone for all `BatchNorm2d` layers during training instead of using the normal `BatchNorm2d` layers. 

Did training get unstable? Like an increasing validation loss after some time due to probably to small batch size such that batch wise mean/std are too different to the running mean/std?

Is there a reason to use FrozenBatchNorm2d instead of turning the BN layers in eval mode?

Thanks in advance and nice work.

**Comment by MasterBin-IIAU** [comment]
Hi, thanks for your appreciation of our work. The reasons why we use ```FrozenBatchNorm2d``` are shown below.
(1) The BN parameters (alpha and beta) and statistics (running mean, running std) pretrained on ImageNet have better generalization ability than those computed on object tracking datasets. 
(2) We found that if we don't use ```FrozenBatchNorm2d``` and use ```nn.BatchNorm2d``` instead, training with DDP will throw out an error. Besides, I think it should not have an obvious performance difference between ```FrozenBatchNorm2d``` and ```nn.BatchNorm2d```.
Thanks.
